_posts/2022-12-01-Stumbling-Into-Greatness.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Stumbling-Into-Greatness"
+date: 2023-03-273T10:00:00
+author: Elizabeth Reich
+layout: post
+categories:
+    - Internships
+tags:
+    - RAC-CCNY Internship Program
+    - Internships
+    - interns
+
+excerpt_separator: <!--more-->
+---
+
+
+
+I was told about an internship opportunity with the Rockefeller Archive Center by my roommate. She encouraged me to apply, knowing that I was looking for new opportunities. I applied, was interviewed, and began to work with the RAC in the Fall of 2022. I had no previous experience in archival work, but was excited nonetheless. I was ready to explore more of what you could do with a History degree. Little did I know that I had stumbled into greatness.
+

```

Comment:
I'd say add the excerpt separator here.

`<!--more-->`

Reply:
@p-galligan Does there need to be a space before/after the separator? Can it be placed right after the paragraph, then add a space before the next paragraph?

Reply:
I'd say an empty line on either side of it for code legibility reasons. Thanks!